pallets/gear/src/benchmarking/code.rs

```diff
@@ -195,7 +195,7 @@ where
         for func in def.imported_functions {
             let sig = builder::signature()
                 .with_params(func.params)
-                .with_results(func.return_type.into_iter().collect())
+                .with_results(func.return_type.into_iter().collect::<Vec<ValueType>>())
```

Comment:
```suggestion
                .with_results(func.return_type.into_iter().collect::<Vec<_>>())
```

Doesn't it catch it?